REVIEWING.md

```diff
@@ -0,0 +1,28 @@
+# Reviewing Yari changes
+
+This document provides information on how to review changes to the yari repo.
+
+## Before you start
+
+Set up the yari repo and the corresponding [content repo](https://github.com/mdn/content) locally, as described in the [Yari quickstart](README.md#quickstart) guide. Once you've got them successfully set up, run the `yarn` and `yarn dev` commands to update your fork with the latest packages and start the MDN test site running locally on `localhost:3000`.
```

Comment:
```suggestion
Set up the Yari repo and the corresponding [content repo](https://github.com/mdn/content) locally, 
as described in the [Yari quickstart](README.md#quickstart) guide. Once you've got them 
successfully set up, run the `yarn` and `yarn dev` commands to update your fork with the 
latest packages and start the MDN test server running locally on `localhost:3000`.
```
Yari, not yari. Lines not too long. server instead of site.

Reply:
fixed, fixed, and fixed